### PR TITLE
putting logout button in li

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by smarzban** [comment]
👍

File: freebasics/templates/skins/base.html [hunk 1]
```diff
@@ -59,16 +59,18 @@
...
-        {% endif %}
         <!-- Tasks Menu -->
         <li class="dropdown tasks-menu">
           <!-- Menu Toggle Button -->
           <a href="#">
             <i class="fa fa-flag-o"></i>
           </a>
         </li>
+        <li>
+          {% if request.user.is_authenticated %}
```
Comment (miltontony) [inline freebasics/templates/skins/base.html:70]: The `if` statement must wrap around the `<li>`

**Comment by hodgestar** [comment]
@codiebeulaine It looks like you merged this without addressing @miltontony's comment about the `if` needing to wrap the `<li>`?

**Comment by codiebeulaine** [comment]
@hodgestar yeah sorry I made a new PR #19  and fixed it :)

**Comment by hodgestar** [comment]
Yay, all good. :)

Although PR #19's name is terribly vague.

**Pull request merged**